operations/mimir-mixin/dashboards/reads.libsonnet

```diff
@@ -169,6 +169,23 @@ local filename = 'mimir-reads.json';
         $.panelDescription(title, description) +
         $.latencyPanel('cortex_query_scheduler_queue_duration_seconds', '{%s}' % $.jobMatcher($._config.job_names.query_scheduler))
       )
+      .addPanel(
+        local title = 'Queue length';
+
+        $.timeseriesPanel(title) +
+        $.panelDescription(title, description) +
+        $.hiddenLegendQueryPanel(
+          'sum(max_over_time(cortex_query_scheduler_queue_length{%s}[$__interval]))' % [$.jobMatcher($._config.job_names.query_scheduler)],
```

Comment:
our alerts use min_over_time

https://github.com/grafana/mimir/blob/938634aea5e16420c5fd0aed30d923c8826cd3cb/operations/mimir-mixin/alerts/alerts.libsonnet#L148

should we use the same for consistency?

Reply:
Good catch, I was experimenting with `max_over_time` to understand the impact of using one or the other, but I forgot to change it back.

Fixed in 53ab3e4.